### DISCUSSION TITLE
Add assets to multiple albums at once

### DISCUSSION THREAD

**Discussion by MD77MD** [immich-app/immich#14291] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently we can only add an asset to multiple albums one at a time... this takes time especially if your adding many assets... because we have to do the hunt and reselect all the assets i want to add for every album i want to add them to.

i wish we can just add assets to multiple albums at once
![SmartSelect_20241119_001017_Vivaldi](https://github.com/user-attachments/assets/622e5d73-013a-4726-8968-54aaa631fb43)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
It's still a duplicate

**Reply by MD77MD** [reply]
how? i made sure that thier isn't a similar request before posting

**Comment by MD77MD** [comment]
@bo0tzz 
why did you close this... there is no duplicate

**Reply by bo0tzz** [reply]
I gave you the duplicate in your previous thread already, #2686
